main/src/ui/conversation_selector/list.vala

```diff
@@ -147,6 +189,21 @@ public class List : ListBox {
         }
     }
 
+    private GLib.List<ListBoxRow> get_visible_rows() {
+        GLib.List<ListBoxRow> visible_rows = new GLib.List<ListBoxRow>();
+        foreach (Widget row in get_children()) {
+            if (filter((ListBoxRow) row))
+                visible_rows.append((ListBoxRow) row);
+        }
+        return visible_rows;
+    }
+
+    private void select_visible_row(uint nr) {
+        GLib.List<ListBoxRow> rows = get_visible_rows();
+        if (rows.length() >= nr)
+            rows.nth(rows.length() - nr).data.activate();
```

Comment:
This makes no sense IMHO: the last rows are those whose last message is most in the past. These are usually those you don't contact rather often, so why put a shortcut on them?